config/initializers/datadog.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
 Datadog.configure do |c|
-  c.tracer(enabled: false) unless Rails.env.production?
+  c.tracing(enabled: false) unless Rails.env.production?
```

Comment:
@lizgarciao can you change `c.tracing(enabled: false) unless Rails.env.production?` to
 `c.tracing.enabled = false unless Rails.env.production?`